docker-compose.yml.template

```diff
@@ -1,20 +1,43 @@
-version: "3.7"
 services:
-  # This is the name of the container
-  # we'll connect to to run commands
-  REPO:
-    image: ACCOUNT/REPO:TAG
+  # This is the name of the container where our code
+  # is going to run. The key can be anything. Note that this
+  # key will be the name of this image on the network that docker-compose
+  # creates, so make it explicit and obvious.
+  %REPO%:
+    # This is the image where our app runs. image: is a keyword. The value
+    # is the name of a Docker image, and likely in the format ACCOUNT/REPO:TAG
+    image: %ACCOUNT%/%REPO%:%TAG%
+    # This tells Docker what to do when starting up. In the case of our app, we don't
+    # want it to pull from a registry unless it's not here.
+    pull_policy: "missing"
```

Comment:
@jgarber623 I used your gist as inspiration for tightening this up.  I have no idea what `init` does, but it does make exiting faster! :)

Reply:
It has something to do with how the container’s command is run (in a separate process or something?). I don’t actually know either. My neck beard isn’t that bushy. 

But yeah, it’s loads faster and I don’t _really_ care why.